templates/page.html

```diff
@@ -244,23 +242,23 @@ <h3>Researchers</h3>
         <p>
           Find out how you can use REANA to describe, run, preserve and reuse your analyses.
         </p>
-        <a href="http://reana.readthedocs.io/en/latest/userguide.html">User Guide</a>
+        <a href="https://docs.reana.io/getting-started/">User Guide</a>
       </div>
       <div class="col">
         <i class="fab fa-mixcloud fa-5x"></i>
         <h3>Administrators</h3>
         <p>
           Install and manage the REANA reusable analysis platform on your own compute cloud.
         </p>
-        <a href="http://reana.readthedocs.io/en/latest/administratorguide.html">Administrator Guide</a>
+        <a href="https://docs.reana.io/development/">Administrator Guide</a>
```

Comment:
There is a bit of discrepancy between how this section and docs.reana.io is organised. There, the development section contains development + admin, that's why the link is the same as the below one.

Reply:
... and we are mixing deploying and developing there, e.g. "Managing users" is something that comes after "Development" while it is clearly more of an admin topic than a developer topic.